src/pages/gen2/reference/amplifyconfiguration/index.mdx

```diff
@@ -16,3 +16,50 @@ In Amplify (Gen 2), the CLI will generate an `amplifyconfiguration.json` file wi
 You can also manually create this file for a specified Amplify app ID and branch, or an AWS CloudFormation stack name with [`amplify generate config`](/gen2/reference/cli-commands#amplify-generate-config).
 
 {/* @TODO full type/interface to reference */}
+
+## Extending Amplify configuration file
+
+The `amplifyconfiguration.json` file is not just a static artifact; it's designed to be extendable to suit the evolving needs of your application. By leveraging the `addOutput` method you can programmatically add or modify configurations. This is particularly useful for customizing outputs that are not directly exposed through the Amplify constructs or for dynamically adjusting your app's configuration in response to changes in your backend strategy.
```

Comment:
1. How will customers find what a `addOutput` method is?
2. Do we allow to modify existing configurations? See https://github.com/aws-amplify/amplify-backend/pull/978#discussion_r1479750371

Reply:
modified the wording to callout a this is a method on the backend and removed override
added a callout about modifying amplify managed resources